robyn/openapi.py

```diff
@@ -0,0 +1,264 @@
+import re
+from dataclasses import asdict, dataclass, field
+from inspect import Signature
+from pathlib import Path
+from typing import Dict, List, Optional, TypedDict
```

Comment:
```suggestion
from typing import Optional, TypedDict
```

We can just use `dict` and `list`